OpenRA.Mods.Cnc/Traits/EdibleByLeap.cs

```diff
@@ -0,0 +1,37 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Traits
+{
+	[Desc("Allows this actor to be the target of an attack leap.")]
+	public class EdibleByLeapInfo : TraitInfo<EdibleByLeap> { }
+
+	public class EdibleByLeap
+	{
+		Actor leaper;
+
+		public bool CanLeap(Actor targeter)
+		{
+			return leaper == null || leaper.IsDead || leaper == targeter;
```

Comment:
Maybe I'm missing something, but why do we return 'true' if the leaper is null or dead?

Shouldn't this be `leaper != null && !leaper.IsDead && leaper == targeter`?